examples/Nat/nat_topo.py

```diff
@@ -113,7 +113,7 @@ def test():
     # Start the Pax NAT process on the NAT node:
     # Start it in a separate terminal so that we can see the output in real time.
     print "Starting Pax NAT process on %s:" % nat0
-    cmd = 'Bin/Pax.exe examples/Nat/nat_wiring.json examples/Bin/Examples.dll'
+    cmd = 'Bin/Pax.exe examples/Nat/nat_wiring_test.json examples/Bin/Examples.dll'
```

Comment:
No, it's just so tests can be completed quicker really






On Tue, Jul 26, 2016 at 11:39 PM +0100, "Nik Sultana" <notifications@github.com> wrote:





> @@ -113,7 +113,7 @@ def test():
>      # Start the Pax NAT process on the NAT node:
>      # Start it in a separate terminal so that we can see the output in real time.
>      print "Starting Pax NAT process on %s:" % nat0
> -    cmd = 'Bin/Pax.exe examples/Nat/nat_wiring.json examples/Bin/Examples.dll'
> +    cmd = 'Bin/Pax.exe examples/Nat/nat_wiring_test.json examples/Bin/Examples.dll'

Out of interest are the two configurations intended to differ other than by timeouts?

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/niksu/pax/pull/12/files/07277e9a85e90c2258009c9fc280847c8a1c0268..ef7115111140c1f4b02c2219b8bfe9cde974bc38#r72351361